src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
+
+  static public class Period {
```

Comment:
Although this has the same name as Joda's Period seems to be a very different concept. Joda's period is a length of time, where as this class is holding the opening and closing times of a Place over a week. Am I misunderstanding your intent?